examples/simple-dg.py

```diff
@@ -233,8 +239,8 @@ def get_inverse_mass_matrix(self, grp, dtype):
         return actx.freeze(actx.from_numpy(matrix))
 
     def inverse_mass(self, vec):
-        if isinstance(vec, np.ndarray):
-            return obj_array_vectorize(self.inverse_mass, vec)
+        if not isinstance(vec, DOFArray):
+            return map_dof_array_container(self.inverse_mass, vec)
```

Comment:
Any reasons to no just add a `@mapped_over_dof_arrays` to the whole thing?

Reply:
The `multi` methods feel sloppy to me. They're also slow, because they have to discover who they array container is. I don't really like them.

Reply:
Fair enough. Should they be removed in favor of a version that requires all arguments to be array containers?

I'll also add a TODO to use this idiom instead in making the connections map over array containers.

Reply:
No need to remove IMO. It's OK to offer them to user code. I'm just wary of using them ourselves. :)